.changeset/violet-walls-see.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sap-ai-sdk/orchestration': minor
+---
+
+[New Functionality] Add support for orchestration translation module. 
+Implement `buildOrchestrationTranslationConfig` convenience function to build translation config.
```

Comment:
```suggestion
Implement `buildOrchestrationTranslationConfig()` convenience function to build translation config.
```

Reply:
[pp] I would rename it as `buildTranslationConfig()` to follow the same naming style as other convenience functions.